test/specs/print_utils/tests/01.print_utils.js

```diff
@@ -0,0 +1,72 @@
+exports.execute = function (options) {
+    var module = options.includes.ermRest;
+    describe("For pretty printing values based on a value's type, ", function () {
+        // Test Cases:
+        it('_printFloat() should format floats correctly.', function () {
+            var _printFloat = module._printFloat;
+            var options = {numDecDigits: 4};
+            expect(_printFloat(null)).toBe('');
+            expect(_printFloat(1234.0, options)).toBe('1,234.0000');
+            expect(_printFloat(23.0)).toBe('23.00');
+            expect(_printFloat(.000)).toBe('0.00');
+        });
+
+        it('_printBoolean() should format booleans correctly.', function () {
+            var _printBoolean = module._printBoolean;
+            expect(_printBoolean(null)).toBe('');
+            expect(_printBoolean(true)).toBe('true');
+            expect(_printBoolean(false)).toBe('false');
+        });
+
+        it('_printInteger() should format integers correctly.', function () {
+            var _printInteger = module._printInteger;
+            expect(_printInteger(null)).toBe('');
+            expect(_printInteger(12341234)).toBe('12,341,234');
+            expect(_printInteger(0001)).toBe('1');
+            expect(_printInteger(233)).toBe('233');
+            expect(_printInteger(23.)).toBe('23');
+            expect(_printInteger(23.000000000)).toBe('23');
+            expect(_printInteger(.0000)).toBe('0');
+            expect(_printInteger(23.101)).toBe('23');
+            expect(_printInteger(.1)).toBe('0');
+        });
+
+        it('_printDate() should format dates correctly.', function () {
+            var _printDate = module._printDate;
+            expect(_printDate(null)).toBe('');
+            expect(_printDate('2012-04-25 13:00:00.00 PST', {
```

Comment:
This testcase if failing on **Travis**. You can check it [here](https://travis-ci.org/informatics-isi-edu/ermrestjs/builds/149269930#L1891).

I see that the build is **green** though one testcase has failed. I have pushed some changes to handle the test suite exit code handling in PR #126 .